### PR TITLE
[WIP] Add babel-standalone with es2015 preset for transpiling

### PR DESCRIPTION
**Pull request by vsashyn**

### CODE REVIEW

**Comment by kof** [comment]
Thanks, so the only missing part is how to use exports, right? Any progress there?

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ node_modules
 npm-debug.log
 tmp
 *~
+dist
```
Comment (kof) [inline .gitignore]: we actually need the dist in the repo because we use github pages

Comment (vsashyn) [reply .gitignore]: Yes, sure, sorry for that. Will fix!

**Comment by kof** [comment]
Btw. an interesting improvement would be to move babel to a webworker!

**Comment by kof** [comment]
Babel itself is doing it right now https://github.com/babel/website/pull/1346

**Comment by vsashyn** [comment]
Yea, cool idea, I see. Will take a look on their progress .  Thank you!

**Comment by kof** [comment]
Closed by accident?

**Comment by vsashyn** [comment]
Yes. `babel-standalone`, which we are going to use was moved to `babel` package. I'm going to add it in a new way and will update PR.

**Pull request closed without merging**